qt_ui/widgets/QTopPanel.py

```diff
@@ -112,6 +112,11 @@ def setGame(self, game: Optional[Game]):
         self.transfers.setEnabled(True)
 
         self.conditionsWidget.setCurrentTurn(game.turn, game.conditions)
+
+        self.conditionsWidget.setToolTip(
+            f"Cloud Base: {game.conditions.weather.clouds.base}m / {round(game.conditions.weather.clouds.base * 3.281)}ft"
```

Comment:
Instead of hardcoding this conversion factor, consider using:

`game.utils.METERS_TO_FEET`

Reply:
![image](https://user-images.githubusercontent.com/47610393/123695206-e4414280-d81f-11eb-9573-417cdccd69c8.png)

regarding this (the pic): won't this break stuff? like i figure .METERS_TO_FEET would be a 'good enough' solution without breaking anything